src/main/java/com/edmunds/rest/databricks/DTO/ClusterEventsDTO.java

```diff
@@ -16,15 +16,13 @@
 
 package com.edmunds.rest.databricks.DTO;
 
+import com.edmunds.rest.databricks.DTO.clusters.ClusterEventDTO;
 import java.util.List;
 import lombok.Data;
-import lombok.Getter;
-import lombok.Setter;
 
 @Data
 public class ClusterEventsDTO {
 
```

Comment:
can you explain why getter and setter annotations are no longer needed?

Reply:
because they're already provided by `@Data` annotation

Reply:
oh ok thanks!